packages/components/src/core/ButtonIcon/index.stories.tsx

```diff
@@ -195,6 +210,250 @@ export const LivePreview = {
+    sdsSize: SDSSizes[number];
+    on: (typeof ON_OPTIONS)[number];
+    onLabelNeeded: boolean;
+  }) {
+    return (
+      <div style={displayContents}>
+        {/* only show the "On: ..." label for combinations that have `on` as a prop */}
+        {onLabelNeeded && (
+          <h5 style={thirdLabel}>
+            On: <b>{on ? "true" : "false"}</b>
+          </h5>
+        )}
+        {DISABLED_OPTIONS.map((disabled) => {
+          return (
+            <ButtonIconDisabledOption
+              sdsType={sdsType}
+              sdsSize={sdsSize}
+              on={on}
+              disabled={disabled}
+              key={String(disabled)}
+            />
+          );
+        })}
+      </div>
+    );
+  }
+
+  // loop through all PSEUDO_STATES + create headers for DISABLED_OPTIONS, PSEUDO_STATES
+  function ButtonIconDisabledOption({
+    sdsType,
+    sdsSize,
+    on,
+    disabled,
+  }: {
+    sdsType: SDSTypes[number];
+    sdsSize: SDSSizes[number];
+    on: (typeof ON_OPTIONS)[number];
+    disabled: (typeof DISABLED_OPTIONS)[number];
+  }) {
+    const SDS_ICONS = {
+      primary: {
+        large: "grid",
+        small: "barChartVertical3",
+      },
+      secondary: {
+        large: "infoSpeechBubble",
+        small: "plusCircle",
+      },
+      tertiary: {
+        large: "xMark",
+        medium: "xMark",
+        small: "xMark",
+      },
+    };
+
+    return (
+      <div style={penultimateLevel}>
+        {PSEUDO_STATES.map((state) => {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore: invalid `sdsIcon` is skipped in <ButtonIconTypeOption />
+          const sdsIcon = SDS_ICONS[sdsType][sdsSize];
+
+          return (
+            <div style={bottomLevel}>
+              {/* remove irrelevant disabled iterations: when combined with all pseudo-states except default, `disabled=false` is impossible */}
+              {(disabled === false ||
+                (disabled === true && state === "default")) && (
+                <>
+                  <h6 style={bottomLabel}>
+                    {disabled === false ? "State: " : "Disabled: "}
+                    <br />
+                    <b>{disabled === false ? state : "true"}</b>
+                  </h6>
+                  <ButtonIcon
+                    aria-label={sdsIcon}
+                    sdsIcon={sdsIcon}
+                    data-testid="button-icon"
+                    sdsType={sdsType}
+                    sdsSize={sdsSize}
+                    on={on}
+                    disabled={disabled}
+                    className={`pseudo-${state}`}
+                    key={state}
+                  />
+                </>
+              )}
+            </div>
+          );
+        })}
+      </div>
+    );
+  }
+
+  // }, // close render
+}; // close ScreenshotTest
+
+export const ScreenshotTest = {
+  parameters: {
+    controls: {
+      exclude: ["disabled", "on", "sdsIcon", "sdsSize", "sdsType"],
```

Comment:
Disables the control panel in the UI since it is not needed on the Chromatic screenshot story page